js/impress.js

```diff
@@ -716,12 +716,19 @@
                     case 40: // down
                              api.next();
                              break;
+                    case 36: // character o
+                    case 79: // home 
+                            var overview = document.getElementById("overview");
+                            if (overview) {
+                                api.goto(overview);
+                            }
+                            break; 
```

Comment:
Thanks! I've changed it now. :)

On Thursday, January 14, 2016, Jacek Kopecký <notifications@github.com>
wrote:

> In js/impress.js
> <https://github.com/impress/impress.js/pull/509#discussion_r49751950>:
>
> > @@ -716,12 +716,19 @@
> >                      case 40: // down
> >                               api.next();
> >                               break;
> > +                    case 36: // character o
> > +                    case 79: // home
> > +                            var overview = document.getElementById("overview");
> > +                            if (overview) {
> > +                                api.goto(overview);
> > +                            }
> > +                            break;
>
> the above could just be api.goto("overview") which will do nothing if
> such an element is not found.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/impress/impress.js/pull/509/files#r49751950>.
>